os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/i2s_api.c

```diff
@@ -20,7 +20,7 @@
 #include "ameba_audio_clock.h"
 #include "amebasmart_i2s.h"
 
-#define SP_MAX_DMA_PAGE_NUM 8
+#define SP_MAX_DMA_PAGE_NUM 9
```

Comment:
Is there any reason why you limit the number?

Reply:
There is no restriction/limit of the page number for i2s dma page, but this page size will be used to initialize static buffer array for i2s dma (larger page number will means bigger binary size), therefore i have changed it to 9 (8 dma page + 1 zero page) so that we can support 8 pages of I2S DMA buffer.
static struct GDMA_CH_LLI LliTx[SP_MAX_DMA_PAGE_NUM];

Reply:
From you reply, it looks it's a configurable value. Then, how about making a new config for this?
Because this is in BSP C code, it's impossible to know.

Reply:
Hi @sunghan-chang,
I have modified this PR to include config for user to set I2S DMA page and size, and the maximum size in bsp is synced with value set by the user. Please help to review this PR again, thank you.